lib/web_ui/test/text_editing_test.dart

```diff
@@ -1903,7 +1903,7 @@ Future<void> testMain() async {
         // TODO(mdebbar): https://github.com/flutter/flutter/issues/50769
         skip: browserEngine == BrowserEngine.edge);
 
-    test('does not send input action in multi-line mode', () {
+    test('sends input action in multi-line mode', () {
```

Comment:
Do we know how many people relied in the old behavior? Is this going to be a breaking change?